cmdeploy/src/cmdeploy/dns.py

```diff
@@ -6,19 +6,22 @@
 from . import remote
 
 
-def get_initial_remote_data(sshexec, mail_domain):
+def get_initial_remote_data(sshexec, mail_domain, iroh_enabled):
     return sshexec.logged(
-        call=remote.rdns.perform_initial_checks, kwargs=dict(mail_domain=mail_domain)
+        call=remote.rdns.perform_initial_checks, kwargs=dict(mail_domain=mail_domain, iroh_enabled=iroh_enabled)
     )
 
 
-def check_initial_remote_data(remote_data, print=print):
+def check_initial_remote_data(remote_data, require_iroh, *, print=print):
     mail_domain = remote_data["mail_domain"]
     if not remote_data["A"] and not remote_data["AAAA"]:
         print(f"Missing A and/or AAAA DNS records for {mail_domain}!")
     elif remote_data["MTA_STS"] != f"{mail_domain}.":
         print("Missing MTA-STS CNAME record:")
         print(f"mta-sts.{mail_domain}.   CNAME  {mail_domain}.")
+    elif require_iroh and remote_data["IROH"] != f"{mail_domain}.":
```

Comment:
```suggestion
    elif require_iroh and remote_data["IROH"] != f"iroh.{mail_domain}.":
```
This needs to compare against the DNS record being `iroh.example.org`, right?